lib/cards.js

```diff
@@ -215,40 +213,43 @@ export const CARD_BRAND = {
    * @returns {Promise<Card[]>}
    * @tag Cards
    */
-   async list(accountID) {
-    checkString(accountID).or(Err.AccountID);
+  async list(accountID) {
+    checkString(accountID).or(Err.MISSING_ACCOUNT_ID);
 
     const result = await this.moov
       .got({
         url: `accounts/${accountID}/cards`,
         method: "GET",
+        json: card,
```

Comment:
@sethlivingston this might not be expected. unless I am missing something 🤔

Reply:
@paulMery Yep that looks like a bug that's been there a while. It's harmless because card is `undefined`, but I'll fix it for the next release. Thanks!